ddl/schema.go

```diff
@@ -135,14 +137,14 @@ func getIDs(tables []*model.TableInfo) []int64 {
 }
 
 func (d *ddl) delReorgSchema(t *meta.Meta, job *model.Job) error {
+	var startKey kv.Key
 	var tableIDs []int64
-	if err := job.DecodeArgs(&tableIDs); err != nil {
-		// arg error, cancel this job.
+	if err := job.DecodeArgs(&tableIDs, &startKey); err != nil {
```

Comment:
I have the same concerns here. We should make sure no this kind of ddl job is running when upgrade.
Or maybe better to add some logic to fallback to table's startKey when it is not set in job.
The rest LGTM.